model/event/subscriptions.go

```diff
@@ -451,6 +453,8 @@ func CreateOrUpdateImplicitSubscription(resourceType string, id string,
 			switch resourceType {
 			case ImplicitSubscriptionPatchOutcome:
 				temp = NewPatchOutcomeSubscriptionByOwner(user, subscriber)
+			case ImplicitSubscriptionPatchFirstFailure:
+				temp = NewFirstTaskFailureInVersionSubscriptionByOwner(user, subscriber)
```

Comment:
Did you double check that this doesn't notify for commits where the user is the author? It looks like the task triggers will match on owner if the task's version was authored by that person

Reply:
By commits do you mean mainline builds vs patches? Will selecting on PatchRequester work with that? (I understand vaguely how the Subscription works but if you have a good description that would also be good)

Reply:
Yep exactly. How it works is actually pretty elegant - https://github.com/evergreen-ci/evergreen/blob/master/model/event/subscriptions.go#L139 will run the triggers for a given file in the triggers package if an event is logged and the user has selectors that are a subset of the selectors for the trigger. So I think what you changed should work